README.md

```diff
@@ -1 +1,33 @@
+# What is the Weather? App
+
+A weather application to demonstrate API calls and Manipulation.
+
+## Install (MAC)
+
+* Install Homebrew ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
+* Install the most current version of node.js ```brew install node```
+* Download this "weather" project file
+* In terminal, navigate to your weather file.
+* Open example.env and follow the directions there.
+* Run the following commands to install dependencies ``npm install axios`` and ``npm install dotenv``
+* Enjoy!
+
+## Usage
+
+Have your terminal opened to the weather directory.
+
+```zsh
+node app.js {your-city},{your-state}
```

Comment:
What if someone omits a comma?
I know that is probably outside the scope of your project itself but it is something to think about especially whenever you are handling user input. In this context your user is a developer so its reasonable to expect them to follow the instructions but if the intended user was less technical you would want lots of validation on the user input.

Reply:
Would validation be setting variables to correct the format in a variety of situations?

Reply:
Would this be done with a conditional for each circumstance?